common/database/repositories/SurveyResultsRepo.ts

```diff
@@ -5,6 +5,8 @@ import Collections from 'common/database/collections';
 type SurveyResults = {
     date: number,
     results: QolSurveyResults,
+    startDate: number,
+    questionCompletionDates: number[],
```

Comment:
Can we do a typedef of number in models/QoL (aliasing it to something like QoLSurveyTimestamp) and use that throughout? It would be nice to have an abstraction on top of the internal time representation in case we change it later.